suite/tests/client-interface/drx-scattergather.dll.c

```diff
@@ -54,9 +54,41 @@ event_exit(void)
     dr_fprintf(STDERR, "event_exit\n");
 }
 
+static dr_emit_flags_t
+event_bb_app2app(void *drcontext, void *tag, instrlist_t *bb, bool for_trace,
+                 bool translating)
+{
+    instr_t *instr;
+    bool expanded;
+    bool scatter_gather_present = false;
+
+    for (instr = instrlist_first(bb); instr != NULL; instr = instr_get_next(instr)) {
+        if (instr_is_gather(instr)) {
+            scatter_gather_present = true;
+        }
+        if (instr_is_scatter(instr)) {
+            scatter_gather_present = true;
+        }
+    }
+    /* Insert a meta instr to test drutil_expand_rep_string() handling it (i#1055) */
+    instrlist_meta_preinsert(bb, instrlist_first(bb), INSTR_CREATE_label(drcontext));
```

Comment:
?  This looks like a mistake.

Reply:
The was copy/paste from drutil_expand_rep_string(), missed updating the comment. But now since we're talking about it, I think this is irrelevant right now to this particular test. I've removed it.